contracts/l1/L1DAIWormholeBridge.sol

```diff
@@ -37,27 +39,6 @@ interface TokenLike {
 }
 
 contract L1DAIWormholeBridge is OVM_CrossDomainEnabled {
-  // --- Auth ---
```

Comment:
I think this bridge needs to be closable by governance (both l1 and l2 counterpart). That's why we need auth... Sorry I forgot to mention this :(

Reply:
Hmm, I agree that being able to block calls to l2DAIWormholeBridge.initiateWormhole() could be nice but I don't see why the l1DAIWormholeBridge should be closable.

Reply:
You're right. L1 part doesn't have to be closable. We want to block issuing new chain messages (similarly that we do in base bridge)